administrator/models/user_profile.py

```diff
@@ -0,0 +1,92 @@
+"""
+    models.user_profile
+
+    This module contains model class for users.
+    """
+from django.contrib.auth.base_user import BaseUserManager
+from django.contrib.auth.models import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin
+from django.contrib.auth.models import UserManager
+from django.core.mail import send_mail
+from django.db import models
+
+from django.utils.translation import ugettext_lazy as _
+
+class UserManager(BaseUserManager):
+    def _create_user(self, email, name, password, **extra_fields):
+        """Creates and saves a User with the given email, name and password."""
+
+        if not email:
+            raise ValueError('Users must have an email address')
+
+        email = self.normalize_email(email)
+        user = self.model(email=email, name=name, **extra_fields)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_user(self, email, name, password, **extra_fields):
+        extra_fields.setdefault('is_superuser', False)
+        return self._create_user(email, name, password, **extra_fields)
+
+    def create_superuser(self, email, name, password, **extra_fields):
+        """Creates and saves a superuser with the given email, name and password."""
```

Comment:
Please, refactor this according to PEP 8.
> The Python standard library is conservative and requires limiting lines to 79 characters (and docstrings/comments to 72).
[https://www.python.org/dev/peps/pep-0008/#maximum-line-length](url)